libraries/BLE/src/BLECharacteristic.cpp

```diff
@@ -628,7 +628,9 @@ void BLECharacteristic::setValue(uint8_t* data, size_t length) {
 		log_e("Size %d too large, must be no bigger than %d", length, ESP_GATT_MAX_ATTR_LEN);
 		return;
 	}
+  m_semaphoreSetValue.take();
```

Comment:
could you please make sure that indentation is correct?

Reply:
Ok I will pr again.